page-manager.js

```diff
@@ -82,6 +84,9 @@ define([
             // Stores a local reference to the dependency loader
             this.dependencyLoader = DependencyLoader.getInst();
 
+            /** Stores a local reference to the middleware handler*/
+            this.middleware = Middleware.getInst();
```

Comment:
I think that this singleton isn't necessary: the `PageManager` module already returns a `PageManager` instance instead of its own definition